ml-agents/mlagents/trainers/settings.py

```diff
@@ -546,6 +551,7 @@ def _set_default_hyperparameters(self):
     threaded: bool = True
     self_play: Optional[SelfPlaySettings] = None
     behavioral_cloning: Optional[BehavioralCloningSettings] = None
+    framework: FrameworkType = FrameworkType.TENSORFLOW
```

Comment:
There are 2 ways to have a pytorch trainer then, with `--torch` or by adding `framework: torch` to the config file.

Reply:
Yep, that's true with every CLI option - it goes into the YAML somehow. That way a run can be specified entirely from a YAML file - this is for cloud training and for reproducibility.